charts/logzio-apm-collector/templates/NOTES.txt

```diff
@@ -0,0 +1,25 @@
+{{- if and (eq .Values.dnsPolicy "None") (not .Values.dnsConfig) }}
+{{- fail "[ERROR] dnsConfig should be provided when dnsPolicy is None" }}
+{{ end }}
+
+{{- if not .Values.configMap.create }}
+[WARNING] "configMap" will not be created and `traceConfig`, `spmConfig` and `serviceGraphConfig` will not take effect.
+{{ end }}
+
+{{- if not .Values.resources }}
+[WARNING] No resource limits or requests were set. Consider setter resource requests and limits for your logzio-apm-collector via the `resources` field.
+{{ end }}
+
+{{- $logLevel := lower .Values.otelLogLevel }}
+{{- if not (or (eq $logLevel "info") (eq $logLevel "warn") (eq $logLevel "error") (eq $logLevel "debug")) }}
+{{ fail "[ERROR] The logzio-apm-collector Chart's `otelLogLevel` must be one of 'info', 'warn', 'error' or 'debug'." }}
+{{- end }}
+
+{{- $region := lower .Values.global.logzioRegion }}
+{{- if not (or (eq $region "us") (eq $region "eu") (eq $region "uk") (eq $region "ca") (eq $region "au")) }}
```

Comment:
Shouldn't we fail the installation if this condition is evaluated?

Reply:
I thought of case where there would be a new region, it would be easier to add the code then add a full custom endpoint. Do you think it would be better to fail?